### PR TITLE
V2.8.0

### PR DESCRIPTION
**Pull request by barryf**

### CODE REVIEW

File: lib/micropublish/server.rb [hunk 1]
```diff
@@ -307,6 +307,25 @@ class Server < Sinatra::Application
       end
     end
 
+    post '/media' do
+      require_session
+
+      unless media_endpoint
```
Comment (jamietanna) [inline lib/micropublish/server.rb:313]: Per other PR comment, we may want to tweak this before the release

Comment (hacdias) [reply lib/micropublish/server.rb:313]: @jamietanna's right. Thanks for noticing that!

Comment (jamietanna) [reply lib/micropublish/server.rb:313]: Sorry I missed it on my first glance!

**Comment by barryf** [comment]
Merged and deployed. Thanks @hacdias for this contribution. It's a really valuable addition.

**Pull request merged**